jieba--current-dir在spacemacs中返回错误的值

在Spacemacs中 `(load-in-progress load-file-name)`会返回`~/.emacs.d/init.el`，而不是jieba.el所在的目录。

我的Spacemacs配置如下，
```elisp
(defconst ruibin-packages
  '(
    (jieba :location local)
    jsonrpc
    )
)
(defun ruibin/init-jsonrpc ()
  (require 'jsonrpc)
)

(defun ruibin/init-jieba ()
  (require 'jieba)
  (jieba-mode)
  )
```
启动时会找不到simple-jieba-server.js。

```
module.js:550
    throw err;
    ^

Error: Cannot find module '/Users/ruibin/.emacs.d/simple-jieba-server.js'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3

Process JIEBA-SERVER stderr finished
```
对Elisp不是很熟悉，搜索了一下没找到更好的定位当前文件的方式，这个问题是不是无解的？

--- a/jieba.el
+++ b/jieba.el
@@ -66,7 +66,7 @@
 
 ;;; Utils
 
-(defun jieba--current-dir ()
+(defconst jieba--current-dir
   (let* ((this-file (cond
                      (load-in-progress load-file-name)
                      ((and (boundp 'byte-compile-current-file)
@@ -74,7 +74,12 @@
                       byte-compile-current-file)
                      (t (buffer-file-name))))
          (dir (file-name-directory this-file)))
-    dir))
+    dir)
+  "Directory of jieba.")
+
+(defun jieba--current-dir ()
+  "Return the directory of jieba."
+  jieba--current-dir)
 
 ;;; Backend Access API
 

